bin/before_install

```diff
@@ -21,6 +21,10 @@ if [ -n "$ACT" ]; then
   curl -o- -L https://yarnpkg.com/install.sh | bash
   echo "$HOME/.yarn/bin" >> $GITHUB_PATH
   echo "$HOME/.config/yarn/global/node_modules/.bin" >> $GITHUB_PATH
+
+  # Install cmake so we can build rugged gem
+  sudo apt-get install -y cmake
```

Comment:
Note that the core repo's before_install already does this: https://github.com/ManageIQ/manageiq/blob/5f3f33dc811fc007ad07a5a6a42cc8cfe79ad3f7/bin/before_install#L18-L20

I'd really like to get something like https://github.com/ManageIQ/manageiq-providers-amazon/pull/744 working, but for now this is required.